### PR TITLE
Implementation for Multi-Tag Writer #164

### PR DESCRIPTION
**Pull request by Git5000**
### Description
Implementation of #164 multi tag writer as discussed in the issue #164. A writer can have more then one (comma separated) tag entry.

Linked issue: #164 

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Code style follows the tinylog standard
- [x] All classes and methods have Javadoc
- [x] Changes are covered by JUnit tests including corner cases, errors, and exception handling
- [x] Maven build works including compiling, tests, and checks (`mvn verify`)
- [x] Changes are committed by a verified email address that is assigned to the GitHub account (https://github.com/settings/emails)

### Documentation

Additions or amendments for the [public documentation](https://github.com/pmwmedia/tinylog/wiki/Documentation):

```markdown
The user manual has to be update to reflect the muilti tag situation.
A tag can be now writer.tag = a,b,c
```

### Agreements

- [x] I agree that my changes will be published under the terms of the [Apache License 2.0](https://github.com/pmwmedia/tinylog/blob/v2.0/license.txt)
- [x] I agree that my GitHub user name will be published in the release notes

### CODE REVIEW

**Comment by Git5000** [comment]
Some additional notes:
- As discussed in the issue thread the new feature breaks tags with commas in. However, it is unlikely that this is used.
- The feature breaks (or fixes) one more thing: Tags are trimmed, that is tags cannot start or end with whitespace or be pure whitesapce tags. Pure whitespace tags where possible before (but hopefully nobody uses something like that)
- If the pull request gets approved one has to change the user manual. Who would do that and how?
- As this pull request changes a core feature it probably should be tested independently of the author of the code

File: tinylog-impl/src/main/java/org/tinylog/core/ConfigurationParser.java [hunk 1]
```diff
@@ -75,8 +75,14 @@ public static Level getGlobalLevel() {
 		List<String> tags = new ArrayList<String>();
 		for (String writerProperty : Configuration.getSiblings("writer").keySet()) {
 			String tag = Configuration.get(writerProperty + ".tag");
-			if (tag != null && !tag.isEmpty() && !tag.equals("-") && !tags.contains(tag)) {
-				tags.add(tag);
+			if (tag != null && !tag.isEmpty() && !tag.equals("-")) {
+				String[] tagArray = tag.split(",");
+				for (String tagArrayItem:tagArray) {
```
Comment (pmwmedia) [inline tinylog-impl/src/main/java/org/tinylog/core/ConfigurationParser.java]: Please put spaces around the colon (code style)

**Comment by pmwmedia** [comment]
Thank you for another great pull request :) I have found only a minor code style violation.  The code is easy to read, well tested, and well documented! What more could I wish for :-)

> * As discussed in the issue thread the new feature breaks tags with commas in. However, it is unlikely that this is used.
> * The feature breaks (or fixes) one more thing: Tags are trimmed, that is tags cannot start or end with whitespace or be pure whitesapce tags. Pure whitespace tags where possible before (but hopefully nobody uses something like that)

I have no concerns to break such strange tags :-D

> * If the pull request gets approved one has to change the user manual. Who would do that and how?

I will update the user manual.

> * As this pull request changes a core feature it probably should be tested independently of the author of the code

Sure! I will test it. The code review looks already very promising.

**Comment by Git5000** [comment]
The code style fix is commited. Please check again.

Changes to the tinylog code base are very easy since the code is well designed, structured and easy to understand. The maven build code style checks find most code style violations before you have to find them (it did not find the colon though).

**Review by pmwmedia — APPROVED** [review]
